### ISSUE TITLE
Cannot run tests locally

### ISSUE THREAD

**Issue by speric** [sublimecoder/advent_of_code#1]
Hi @sublimecoder!

I checked out your solution locally, and ran the tests as per the `README`. I got the following:

```
$ ruby test/path_test.rb
Run options: --seed 29184

# Running:

EE

Finished in 0.047746s, 41.8886 runs/s, 0.0000 assertions/s.

  1) Error:
PathTest#test_first_revisited_location:
NoMethodError: undefined method `sum' for [1, 164]:Array
    /Users/ericfarkas/dev/code-challenges/advent_of_code/lib/path.rb:29:in `distance_traversed_for'
    test/path_test.rb:25:in `test_first_revisited_location'

  2) Error:
PathTest#test_shortest_distance:
NoMethodError: undefined method `sum' for [157, 150]:Array
    /Users/ericfarkas/dev/code-challenges/advent_of_code/lib/path.rb:29:in `distance_traversed_for'
    /Users/ericfarkas/dev/code-challenges/advent_of_code/lib/path.rb:21:in `shortest_distance'
    test/path_test.rb:20:in `test_shortest_distance'

2 runs, 0 assertions, 0 failures, 2 errors, 0 skips
```

You're calling `sum` on an Array (https://github.com/sublimecoder/advent_of_code/blob/master/lib/path.rb#L29), and that method is not actually part of the Ruby stdlib for all Ruby versions. It _is_ a Rails extension to `Enumerable` (http://api.rubyonrails.org/classes/Enumerable.html#method-i-sum).

Can you think of a few ways around this, so that the tests (and the solution) will pass locally?

**Comment by speric** [comment]
I am on `ruby 2.3.4p301` locally.

**Comment by sublimecoder** [comment]
Sum was implemented in ruby 2.4.0+ I'm currently using 2.4.2 before 2.4 it required active support to monkey patch enumeralbe to inlclude sum. 

array.inject(0, :+) should be an acceptable substitute. And compatible with several versions back in ruby. I'm pretty sure active support uses inject under the hood.

**Comment by sublimecoder** [comment]
Just fixed the code and tested it against 2.3.4. Should fix the issue for you locally.

**Comment by speric** [comment]
👍
